docker-initunlocklnd.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+echo "[lnd_unlock] Waiting 2 seconds for lnd..."
+sleep 2;
+
+LND_DIR="/root/.lnd"
+CA_CERT="$LND_DIR/tls.cert"
+
+LND_WALLET_DIR="$LND_DIR/data/chain/$1/$2/"
+WALLET_FILE="$LND_WALLET_DIR/wallet.db"
+LNDUNLOCK_FILE=${WALLET_FILE/wallet.db/walletunlock.json}
+MACAROON_FILE="$LND_WALLET_DIR/admin.macaroon"
+MACAROON_HEADER="r0ckstar:dev"
+if [ -f "$MACAROON_FILE" ]; then
+    MACAROON_HEADER="Grpc-Metadata-macaroon:$(xxd -ps -u -c 1000 "$MACAROON_FILE")"
+fi 
+
+if [ -f "$WALLET_FILE" ]; then
+    if [ ! -f "$LNDUNLOCK_FILE" ]; then
+        echo "[lnd_unlock] WARNING: UNLOCK FILE DOESN'T EXIST! MIGRATE LEGACY INSTALLATION TO NEW VERSION ASAP"
+        exit 1
```

Comment:
this is not an error, would send `exit 0`

Reply:
ACK